screen-weather-get.py

```diff
@@ -160,13 +161,15 @@ def main():
         'HOUR_NOW': datetime.datetime.now().strftime("%-I %p"),
         'DAY_ONE': datetime.datetime.now().strftime("%b %-d, %Y"),
         'DAY_NAME': datetime.datetime.now().strftime("%A"),
+        'ALERT_MESSAGE_VISIBILITY': "visible" if alert_message else "hidden",
         'ALERT_MESSAGE': alert_message
     }
 
     logging.debug("main() - {}".format(output_dict))
 
     logging.info("Updating SVG")
-    template_svg_filename = 'screen-template.svg'
+    
+    template_svg_filename = f'screen-template.{template_name}.svg'
```

Comment:
Future templates can be added, they'll just be screen-template.5.svg... 6.svg... and so on.